### ISSUE TITLE
Feature: showVersion function

### ISSUE THREAD

**Issue by OsmanAltun** [yargs/yargs#1829]
**Description:**
A function which has the same signature as the showHelp function. 
Ability to print version information or execute `callback(version: string)`.

**Comment by bcoe** [comment]
@OsmanAltun just curious what your use case is 😄 (_not a no to the feature_).

**Comment by OsmanAltun** [comment]
```js
yargs.version(`Core: ${CoreVersion}, CLI: ${CliVersion}, ...`)
yargs.command({command: 'version', handler: yargs.showVersion, ...})
```
instead of
```js
let version = `Core: ${CoreVersion}, CLI: ${CliVersion}, ...`
yargs.version(version)
yargs.command({command: 'version', handler: () => console.log(version), ...})
```
A way to use version string without having to save it in a variable first. Not that big of a deal, but the flexibility to do as you please is nice (like commands being an alias of command function). The PR I made with bugfixes is WAY more important than this, cuz I need workarounds for those.

**Comment by bcoe** [comment]
```
yargs.version(`Core: ${CoreVersion}, CLI: ${CliVersion}, ...`)
yargs.command({command: 'version', handler: yargs.showVersion, ...})
```

☝️ this works? I'd never thought pf passing a method from yargs itself as a handler.

---


We already treat the `help` command as an alias for `--help`, if we were to add a `version` command by default, would we still need this functionality?

**Comment by OsmanAltun** [comment]
For my case, I also specify the help command explicitly, because the command needs to be present in the help output. It would  be nice if there was a way to include the commands in the help output (because I didn't know that this was a thing, and so does the end user probably lol). However, what you mentioned would be a nice addition.

The project I am working on also has listeners for errors. When an error is thrown, the handler includes version information in the  output and currently there is no way of retrieving this back from yargs. <hr/>

Anyway, I double checked and what I wrote on a whim (`handler: yargs.showVersion`) does not work because an object is given as argument. For now,  `() => yargs.showversion()` works fine. If instead of throwing an error, it would default to `level='log'` or something, what I wrote would be possible though.
